spec/support/vcr.rb

```diff
@@ -1,21 +1,21 @@
 VCR.configure do |c|
```

Comment:
I am unsure how to count the number of cassettes since right now everything is in the same file with the "new" way of doing it. 

I would say that there are about 30 external requests being made spread out through all of our requests specs.